.build/start.ps1

```diff
@@ -11,10 +14,18 @@ param
     [string]
     $Version,
 
+    [Parameter()]
+    [int]
+    $BuildNumber,
+
     [Parameter()]
     [ValidateSet('alpha', 'preview')]
```

Comment:
Should we have a `dev` option for local dev builds to distinguish them from prod builds?

Reply:
Mainly just to differentiate a version that you deployed in dev mode before it was finished. For instance, we ship 0.1, so we update the next version to 0.2, even tho it's not done yet. We'll keep working with that until we're ready to ship and finally move from a dev to production build. I don't know if it's critical, but I've hit this a few times as I've run local deployments on different versions. We can also figure this out later. Just something to mull over.